Sources/UINotificationPresentationContext.swift

```diff
@@ -71,10 +71,10 @@ public final class UINotificationPresentationContext {
         }
         
```

Comment:
<!--
  0 failure: 
  1 warning:  Lines should not ...
  
  
  DangerID: danger-id-Danger;
  File: Sources/UINotificationPresentationContext.swift;
  Line: 72;
-->

- :warning: Lines should not have trailing whitespace. (`trailing_whitespace`)

Reply:
So weird that this project has SwiftLint setup, but also plenty of lines that validate this `trailing_whitespace` rule. Why weren't these fixed when SwiftLint was added?

Also the SwiftLint integration is broken, at least for me: #66